website/docs/d/eventgrid_system_topic.html.markdown

```diff
@@ -40,11 +40,13 @@ The following attributes are exported:
 
 * `source_arm_resource_id` - The ID of the Event Grid System Topic ARM Source.
 
-* `topic_type` - The Topic Type of the Event Grid System Topic. Possible values are: `Microsoft.AppConfiguration.ConfigurationStores`, `Microsoft.Communication.CommunicationServices`
+* `topic_type` - The Topic Type of the Event Grid System Topic. Some possible values are: `Microsoft.AppConfiguration.ConfigurationStores`, `Microsoft.Communication.CommunicationServices`
```

Comment:
We can actually remove all this since it's a datasource and we only care about the description of the attribute and not what the attribute could be. 

```
* `topic_type` - The Topic Type of the Event Grid System Topic.
```

Reply:
Yup, makes sense.